feature-libs/cart/import-export/assets/translations/en/import-export.i18n.ts

```diff
@@ -33,7 +33,10 @@ export const importEntriesDialog = {
     loading: 'Products are being processed... ({{ count }}/{{ total }})',
     successes:
       '{{ successesCount }} out of {{ total }} products have been imported successfully.',
-    problems: '{{ problemsCount }} products were not imported.',
+    problems: '{{ count }} product was not imported totally.',
+    problems_plural: '{{ count }} products were not imported totally.',
```

Comment:
and `warnings` instead of `problems_plural`

Reply:
`_plural` suffix is intentionally (a feature of i18n) https://sap.github.io/spartacus-docs/i18n/#special-parameter-count